app/models/user.rb

```diff
@@ -11,6 +11,9 @@ class User < ActiveRecord::Base
 
   before_create :setup_role
 
+  # add scope
+  scope :comment_notifiable, ->(conference) {joins(:roles).where('roles.name IN (?)', [:organizer, :cfp]).where('roles.resource_type = ? AND roles.resource_id = ?', 'Conference', conference.id)}
```

Comment:
Line is too long. [194/80]<br>Space missing inside {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.